File: C/ex1010.c
#include <stdio.h>

int main()
{
    int     code, unit;
    double  price_unit, price_total;
    scanf("%d %d %lf", &code, &unit, &price_unit);
    price_total = price_unit * unit;
    scanf("%d %d%lf", &code, &unit, &price_unit);
    price_total += price_unit * unit;
    printf("VALOR A PAGAR: R$ %.2lf\n", price_total);
    return 0;
}

File: GO/ex1010.go
package main

import (
	"fmt"
)

func main10() {
	var code int
	var unit int
	var price_unit float64
	var price_total float64

	fmt.Scanf("%d %d %f", &code, &unit, &price_unit)
	price_total = price_unit * float64(unit)
	fmt.Scanf("%d %d %f", &code, &unit, &price_unit)
	price_total += price_unit * float64(unit)
	fmt.Printf("VALOR A PAGAR: R$ %.2f\n", price_total)
}
